.github/workflows/ph_backward_compatibility.yaml

```diff
@@ -12,10 +12,176 @@ defaults:
+          token: ${{github.token}}
+      - if: ${{ matrix.release == '3.1' }}
+        name: Download Prev Leap Version (v3.1.x)
+        uses: AntelopeIO/asset-artifact-download-action@v2
+        with:
+          owner: AntelopeIO
+          repo: leap
+          file: 'leap-.*-${{matrix.platform}}.04-x86_64.deb' # Ex. leap-3.1.4-ubuntu20.04-x86_64.deb
+          target: '${{matrix.release}}'
+          token: ${{github.token}}
+      - name: Extract and Place Rev Leap Version artifacts
+        run: |
+          mkdir tmp
+          dpkg -x leap*.deb tmp
+          rm build/bin/nodeos
+          rm build/bin/cleos
+          mv tmp/usr/bin/nodeos build/bin
+          mv tmp/usr/bin/cleos build/bin
+      - name: Run BP Op Mode Performance Test
+        if: always()
+        run: |
+          cd build
+          ctest --output-on-failure -R performance_test_bp --timeout 480
+      - name: Run CPU Trx Spec Performance Test
+        if: always()
+        run: |
+          cd build
+          ctest --output-on-failure -R performance_test_cpu_trx_spec --timeout 480
+      - name: Run API Node Op Mode Performance Test
+        if: always()
+        run: |
+          cd build
+          ctest --output-on-failure -R performance_test_api --timeout 480
+      - name: Run Read Only Trxs Performance Test
+        if: always()
+        run: |
+          cd build
+          ctest --output-on-failure -R performance_test_read_only_trxs --timeout 480
+      - name: Run P2P Performance Basic Test
+        if: always()
+        run: |
+          cd build
+          ctest --output-on-failure -R performance_test_basic_p2p --timeout 480
+      - name: Run User Defined Transfer Trx Spec Performance Basic Tests
+        if: always()
+        run: |
+          cd build
+          ctest --output-on-failure -R performance_test_basic_transfer_trx_spec --timeout 480
+      - name: Run User Defined New Acct Trx Spec Performance Basic Tests
+        if: always()
+        run: |
+          cd build
+          ctest --output-on-failure -R performance_test_basic_new_acct_trx_spec --timeout 480
+      - name: Run User Defined CPU Trx Spec Performance Basic Tests
+        if: always()
+        run: |
+          cd build
+          ctest --output-on-failure -R performance_test_basic_cpu_trx_spec --timeout 480
+      - name: Run User Defined Ram Trx Spec Performance Basic Tests
+        if: always()
+        run: |
+          cd build
+          ctest --output-on-failure -R performance_test_basic_ram_trx_spec --timeout 480
+      - name: Run API Node Op Mode Performance Basic Test
+        if: always()
+        run: |
+          cd build
+          ctest --output-on-failure -R performance_test_basic_http --timeout 480
+      - name: Run Read Only Trx Performance Basic Test
+        if: always()
         run: |
-          echo "Workflow Stub"
+          cd build
+          ctest --output-on-failure -R performance_test_basic_read_only_trxs --timeout 480
```

Comment:
Having all of these manually defined like this is at risk to forgetting to add one later. Maybe we should have a new label for performance tests, or maybe just `-R performance_test` is enough.

Reply:
Collapsing these back down into a regular expression is the plan. Was splitting them out to get clearer information when a test failed during initial debugging.

Reply:
I believe I can do this with a mix of `-R` and `-E` depending on `nodeos` version.

Reply:
Addressed: 800a37e60